ArmRobot/Assets/Scripts/Camera/RandomPositionCamera.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomPositionCamera : MonoBehaviour
+{
+    public float changePositionCamera = 0.005f;
+
+    // Update is called once per frame
+    void Update()
```

Comment:
Rather than tying the camera repositioning to Update(), which gets called every frame, it might be better to trigger all the domain randomization methods each time we save a datapoint.